core/app/modules/kit-library/assets/js/components/envato-promotion.js

```diff
@@ -13,7 +14,19 @@ export default function EnvatoPromotion() {
 				target="_blank"
 				rel="noreferrer"
 				text={ __( 'Check out Elementor Website Kits on ThemeForest', 'elementor' ) }
+				onClick={ () => eventTrackingDispatch(
+					'kit-library/check-kits-on-theme-forest',
+					{
+						event: 'browse themeforest',
+						source: 'home page',
+						category: '/' === category ? 'all kits' : 'favorites',
```

Comment:
What if there is no category prop at all?

Reply:
In this scenario, at the moment, the category param will always return a value, we can maybe change the condition so it will return 'all kits' by default